tools/add_triage_signature.py

```diff
@@ -50,15 +53,27 @@ def format_description(failure_data):
 def days_ago(datestr):
     try:
         return (datetime.now() - dateutil.parser.isoparse(datestr)).days
-    except:
+    except Exception:
         logger.debug("Cannot parse date: %s", datestr)
         return 9999
 
+@functools.lru_cache(maxsize=None)
```

Comment:
Maybe we should put some limit on the cache? To make sure we don't have any chance to full the RAM if someone will run it on the same Python session